### PR TITLE
[10.0][MIG] web_ir_actions_act_window_message

### PR DESCRIPTION
**Pull request by benwillig**
This PR is based on #611.

Added HTML handling.

### CODE REVIEW

**Review by sbidoul — APPROVED** [review]

**Comment by sbidoul** [comment]
@benwillig could you add a word about the is_html in the readme ?

File: web_ir_actions_act_window_message/static/src/js/web_ir_actions_act_window_message.js [hunk 1]
```diff
@@ -56,10 +55,10 @@ odoo.define('web.web_ir_actions_act_window_message', function (require) {
             {
                 return {
                     text: button_definition.name || 'No name set',
-                    oe_link_class: button_definition.oe_link_class ||
+                    classes: button_definition.oe_link_class ||
```
Comment (zakiuu) [inline web_ir_actions_act_window_message/static/src/js/web_ir_actions_act_window_message.js]: @benwillig shouldn't it be button_definition.classes?

**Comment by benwillig** [comment]
@sbidoul I already added following:

```python
 # Use HTML instead of text
'is_html_message': True,
```

Should I add more explanations?

**Comment by sbidoul** [comment]
@benwillig sorry I had not seen it.

OTOH, there is another issue: you should apply [this procedure](https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-10.0) to forward port the 9.0 version while preserving the commit authors.

File: web_ir_actions_act_window_message/static/src/js/web_ir_actions_act_window_message.js (hunk 1 above)
Comment (benwillig) [reply web_ir_actions_act_window_message/static/src/js/web_ir_actions_act_window_message.js]: @zakiuu  Yes indeed, but I think you commented on an outdated version. Anyway, I used the OCA migration workflow, everything should be ok now.

**Comment by benwillig** [comment]
@sbidoul I used given procedure. I think it's ok now. Sorry for the inconvenience caused.

**Review by zakiuu — APPROVED** [review]
looks good

**Comment by sbidoul** [comment]
looks good to me too :) 

I would have been nice to split is_html in another commit than the migration, but not a blocker (`git add -p` is a great tool to do such kind of commit splitting)

**Comment by pedrobaeza** [comment]
What about #605 ?

**Comment by jesusVMayor** [comment]
:+1:  
cc: @Yajo

**Review by jesusVMayor — APPROVED** [review]

**Pull request merged**